sheetstorm/tests/test_unit_generate_test_data.py

```diff
@@ -0,0 +1,178 @@
+from django.test import TestCase
+from django.utils import timezone
+from parameterized import parameterized
+
+from managers.models import Project
+from sheetstorm.management.commands.generate_test_data import Command
+from sheetstorm.management.commands.generate_test_data import ProjectType
+from sheetstorm.management.commands.generate_test_data import UnsupportedProjectTypeException
+from users.models import CustomUser
+
+
+class CustomCommandOptions:
+    number_of_projects_to_create = 1
+    number_of_users_to_create = 1
+    OPTIONS = {
+        "verbosity": 1,
+        "settings": None,
+        "pythonpath": None,
+        "traceback": False,
+        "no_color": False,
+        "force_color": False,
+        "skip_checks": False,
+        "ADMIN": None,
+        "EMPLOYEE": None,
+        "MANAGER": None,
+        "SUPERUSER": False,
+        "SUSPENDED": number_of_projects_to_create,
+        "ACTIVE": number_of_projects_to_create,
+        "COMPLETED": number_of_projects_to_create,
+    }
+
+
+class CreateUserMethodsTests(TestCase, CustomCommandOptions):
```

Comment:
In projects-related tests you are testing for quantity computation based on objects that already exist in the database, but I can't see any counterpart here.

Also, correct me if I'm mistaken, but I believe that at least one superuser was meant to always exist after the method is run? So why not test that? What if there are no superusers in the database. What if there are superusers in the database? What happens if we specify the number of superusers in both of those cases?

And what if we set a number of users that's lower than the existing quantity? I think that's also worth testing.

Reply:
To add superuser to database required positional argument. There is no possibility to specify number of superusers to create, so only one superuser is created by default, but only if you add argument and superuser not exists in the database. 
I added test that you mentioned at the end. :)